src/views/check-your-answers.njk

```diff
@@ -23,7 +23,7 @@
         {{
           govukSummaryList({
             classes: 'govuk-!-margin-bottom-0',
-            attributes: { id: 'director-details-' + loop.index0 },
+            attributes: { id: 'director-details' },
```

Comment:
This will cause accessibility tests to fail. If 2 directors are selected, then a duplicate ID will be present on the page.

Reply:
This should not have changed